optim.py

```diff
@@ -104,43 +99,39 @@ def validate(model, val_loader, val_writer, criterion, epoch):
     model.eval()
 
     # start iterator with actual sales from previous day
+    val_loader = iter(val_loader)
     day, items, t = next(val_loader)
-    seq_len = day.shape[1]
 
     with torch.no_grad():
         # initialize sales and targets columns for current day
         y = model(day, items)
-        sales = y[..., 0].view(1, -1)
-        targets = t[..., 0].view(1, -1)
+        sales = y[:, :1]
+        targets = t[..., :1]
 
         for day, items, t in tqdm(val_loader, desc=f'Validation Epoch {epoch}'):
-            # DONE - check?
-            # REMARK: sales are not unit figures
-            items[0, -len(sales):, 2] = sales[-seq_len:]
+            # replace actual sales in items with projected sales
```

Comment:
... in items with last seq_len projected sales

Reply:
Nu dus niet meer. Omdat we de data nu in blokken krijgen i.p.v. een sliding window is `seq_len` altijd gelijk aan 1. Deze regel wordt dan een stuk simpler.